test/compose.js

```diff
@@ -65,4 +65,14 @@ describe('compose', function() {
     assert.deepEqual(g(1, 2, 3), [1, 2, 3]);
   });
 
+  it('returns a curried function', function() {
+    function f(a) { return a + '_END'; }
+    function g(b, c, d) { return [b, c, d].join('_'); }
+    var c = R.compose(f, g);
```

Comment:
It's typical to use `g ∘ f` in examples. Let's update the function names so this can read `R.compose(g, f)`.